docs/reference/search/search-your-data/retrieve-selected-fields.asciidoc

```diff
@@ -6,72 +6,46 @@
-response, you can use the `fields` parameter:
+provided when indexing the document. There are two recommended methods to
+retrieve selected fields from a search query:
 
-[source,console]
-----
-POST my-index-000001/_search
-{
-  "query": {
-    "match": {
-      "message": "foo"
-    }
-  },
-  "fields": ["user.id", "@timestamp"],
-  "_source": false
-}
-----
-// TEST[setup:my_index]
-
-The `fields` parameter consults both a document's `_source` and the index
-mappings to load and return values. Because it makes use of the mappings,
-`fields` has some advantages over referencing the `_source` directly: it
-accepts <<multi-fields, multi-fields>> and <<alias, field aliases>>, and
-also formats field values like dates in a consistent way.
-
-A document's `_source` is stored as a single field in Lucene. So the whole
-`_source` object must be loaded and parsed even if only a small number of
-fields are requested. To avoid this limitation, you can try another option for
-loading fields:
-
-* Use the <<docvalue-fields, `docvalue_fields`>>
-parameter to get values for selected fields. This can be a good
-choice when returning a fairly small number of fields that support doc values,
-such as keywords and dates.
-* Use the <<request-body-search-stored-fields, `stored_fields`>> parameter to
-get the values for specific stored fields (fields that use the
-<<mapping-store,`store`>> mapping option).
-
-You can also use the <<script-fields,`script_field`>> parameter to transform
-field values in the response using a script.
-
-You can find more detailed information on each of these methods in the
-following sections:
+* Use the <<search-fields-param,`fields` option>> to extract the values of
+fields present in the index mapping
+* Use the <<source-filtering,`_source` option>> if you need to access the original data that was passed at index time
 
-* <<search-fields-param>>
-* <<docvalue-fields>>
-* <<stored-fields>>
-* <<source-filtering>>
-* <<script-fields>>
+You can use both of these methods, though the `fields` option is preferred
+because it consults both the document data and index mappings. In some
+instances, you might want to use <<field-retrieval-methods,other methods>> of
+retrieving data.
 
 [discrete]
 [[search-fields-param]]
-=== Fields
-
+=== The `fields` option
 // tag::fields-param-desc[]
-The `fields` parameter allows for retrieving a list of document fields in
-the search response. It consults both the document `_source` and the index
-mappings to return each value in a standardized way that matches its mapping
-type. By default, date fields are formatted according to the
-<<mapping-date-format,date format>> parameter in their mappings. You can also
-use the `fields` parameter to retrieve <<runtime-retrieving-fields,runtime field
-values>>.
+To retrieve specific fields in the search response, use the `fields` parameter.
+Because it accesses the index mappings, the `fields` parameter provides several
```

Comment:
Small comment, maybe 'consults the index mappings' or 'makes use of the mappings' would be more descriptive?

Reply:
I like "consults the index mappings" 👍